packages/lib/passport-interface/src/PODPipeline/coercion.ts

```diff
@@ -0,0 +1,82 @@
+import { POD_INT_MAX, POD_INT_MIN, checkBigintBounds } from "@pcd/pod";
+import { SafeParseReturnType, z } from "zod";
+import { PODPipelineInputFieldType } from "../genericIssuanceTypes";
+import { FieldTypeToJavaScriptType } from "./Input";
+
+/**
+ * These coercions are used to convert input values from a CSV document into
+ * the types that we use in the POD pipeline.
+ *
+ * These types will subsequently be converted into {@link PODValue}s, but
```

Comment:
Thoughts?

Reply:
They're just the POD value types plus types used by tickets (date/timestamp, UUID). I might have been too flexible in assuming that there could be cases in which we want to do a second conversion from the parsed input data into some different POD type, which I now think we probably do not need to do.

The flexibility may turn out to be useful for PODTicketPCDs though, which do not (in their public interface) use POD types, but instead JavaScript types, and so we may need additional mappings into PODTicketPCD types.

Reply:
I think this can probably be simplified down to every possible "input type" converting to exactly one PODValue type.  They might be 1:1, or there may be more input types in future (e.g. maybe you someday want an email address type to enforce that it parses correctly, but it's always a string in a POD).

I don't understand what you think is uniquely needed by PODTickets, but maybe I will when I get to that review.  I think the auto conversions should be reversible, at least when you know which PODTicket field you're filling in.